frontend/common/src/helpers/nameUtils.tsx

```diff
@@ -76,3 +77,74 @@ export const getFullNameHtml = (
   }
   return `${firstName} ${lastName}`;
 };
+
+/**
+ * Wraps common abbreviations in abbr tags to make them more accessible
+ *
+ * @param text   text to wrap
+ * @param intl   react-intl object
+ * @param title  abbreviation title
+ */
+export const wrapAbbr = (
+  text: React.ReactNode,
+  intl: IntlShape,
+  title?: string,
+): JSX.Element => {
+  const stringifyText = text && text.toString(); // grabs text from React.ReactNode (is there a better way to get text from React.ReactNode type?)
+  if (typeof stringifyText !== "string") {
+    const fallbackTitle = intl.formatMessage({
+      id: "MuWdei",
+      defaultMessage: "Abbreviation not found.",
+      description:
+        "Message shown to user when the abbreviation text is not found.",
+    });
+    return (
+      <abbr title={fallbackTitle}>
+        <span>{text}</span>
+      </abbr>
+    );
+  }
+  switch (stringifyText) {
+    // Regex that matches with all IT(en)/TI(fr) classification with levels
+    case stringifyText.match(/[IT][TI]-0\d/)?.input:
+      return (
+        <abbr title={intl.formatMessage(getAbbreviations("IT"))}>
+          <span
+            aria-label={stringifyText.replace("-0", "").split("").join(" ")}
```

Comment:
You've used `.split("").join(" ")` a lot.  Maybe worth factoring out into a descriptive util class?  I'm not sure what you would call it.  expand?  expandWith?